eng/Build.ps1

```diff
@@ -611,9 +613,11 @@ try {
         TestUsingMSBuild -testProject "$RepoRoot\tests\FSharp.Build.UnitTests\FSharp.Build.UnitTests.fsproj" -targetFramework $script:desktopTargetFramework -testadapterpath "$ArtifactsDir\bin\FSharp.Build.UnitTests\"
         TestUsingMSBuild -testProject "$RepoRoot\tests\FSharp.Core.UnitTests\FSharp.Core.UnitTests.fsproj" -targetFramework $script:desktopTargetFramework -testadapterpath "$ArtifactsDir\bin\FSharp.Core.UnitTests\"
 
-        # Collect output from  background jobs
-        Wait-job $bgJob | out-null
-        Receive-Job $bgJob -ErrorAction Stop
+        Receive -job $bgJob
+    }
+
+    if ($testDesktop -and -not $ci ) {
+        TestUsingMSBuild -testProject "$RepoRoot\FSharp.sln" -targetFramework $script:desktopTargetFramework -testadapterpath "$ArtifactsDir\bin\FSharp.Compiler.ComponentTests\"
```

Comment:
Why component tests here?

Reply:
I had to put something there. I guess any other test project or just `ArtifactsDir` would do. IIUC this is a search path for .TestAdapter.dll

Reply:
TBH I don't think this does anything useful. According to the docs 

> --test-adapter-path <ADAPTER_PATH>
> 
> Path to a directory to be searched for additional test adapters. Only .dll files with suffix .TestAdapter.dll are inspected. If not specified, the directory of the test .dll is searched.

It seems we always specify more or less the .dll folder, which is searched by default.

Reply:
I think this might have stayed here since the times of NUnit presence in the repo. Or maybe either xunit versions were relying on this... Either way, yes, this totally might be redundant now.